lib/chartlib.sh

```diff
@@ -52,6 +53,11 @@ echo " CHART_YAML_SCHEMA=$CHART_YAML_SCHEMA"
 echo " VALIDATE_MAINTAINERS=$VALIDATE_MAINTAINERS"
 echo " GITHUB_INSTANCE=$GITHUB_INSTANCE"
 echo " CHECK_VERSION_INCREMENT=$CHECK_VERSION_INCREMENT"
+if [ -n "$HELM_EXTRA_OPTS" ]; then
```

Comment:
Be consistent. Just print it as for the other vars.

Reply:
For consistency, make it an array.

Reply:
`helm test` and `helm delete` - your're right - totally forgot about that (facepalm).

About printing HELM_EXTRA_OPTS in console:
For security it was not set to be printed in the console, because someone could pass custom var override (such as password/key) - of course this could be mitigated by ensuring someone is generating full .yaml and then uses `-f` flag to override those values.

About converting HELM_EXTRA_OPTS into array:
This is explained in the expanded README.md Making it as plain string and not an array was done on purpose - not everyone is on bash 4.4 yet, also I have not checked if you can easily pass bash array as env var into the docker container. See #3

Reply:
Arrays are used already and Bash 4.4 is required anyways.